department-of-veterans-affairs/caseflow
# Case Review | Allow judges to rate the quality of OMO requests

Resolves https://github.com/department-of-veterans-affairs/caseflow/issues/7157

### Description
Changes the OMO assignment flow to require the judge to rate the quality of OMO requests.

### Testing Plan
- Become BVASCASPER1.
- Create an OMO request for a legacy case with document ID "12345-12345678" and assign it to "Judge with hearings and cases" (BVAAABSHIRE).
- Become BVAAABSHIRE.
- Open the case.
- Select "Assign OMO".
- Fill out the evaluation form.
- Click "Continue".
- Confirm that the success message mentions OMO.
